rwengine/src/data/AnimGroup.cpp

```diff
@@ -1,39 +1,390 @@
+         {"woman_run", 0xe2},
+         {"woman_runpanic", 0xe2},
+         {"woman_idlestance", 0x2},
+     }},
+    {"shopping",
+     {
+         {"woman_walkshop", 0xe2},
+         {"woman_run", 0xe2},
+         {"woman_run", 0xe2},
+         {"woman_idlestance", 0x2},
+     }},
+    {"busywoman",
+     {
+         {"woman_walkbusy", 0xe2},
+         {"woman_run", 0xe2},
+         {"woman_runpanic", 0xe2},
+         {"woman_idlestance", 0x2},
+     }},
+    {"sexywoman",
+     {
+         {"woman_walksexy", 0xe2},
+         {"woman_run", 0xe2},
+         {"woman_runpanic", 0xe2},
+         {"woman_idlestance", 0x2},
+     }},
+    {"oldwoman",
+     {
+         {"woman_walkold", 0xe2},
+         {"woman_run", 0xe2},
+         {"woman_runpanic", 0xe2},
+         {"woman_idlestance", 0x2},
+     }},
+    {"fatwoman",
+     {
+         {"walk_fat", 0xe2},
+         {"woman_run", 0xe2},
+         {"woman_runpanic", 0xe2},
+         {"woman_idlestance", 0x2},
+     }},
+    {"panicchunky",
+     {
+         {"run_fatold", 0xe2},
+         {"woman_runpanic", 0xe2},
+         {"woman_runpanic", 0xe2},
+         {"idle_stance", 0x2},
+     }},
+    {"playerback",
+     {
+         {"walk_player_back", 0xe2},
+         {"run_player_back", 0xe2},
+         {"run_player_back", 0xe2},
+         {"idle_stance", 0x2},
+         {"walk_start_back", 0x40},
+     }},
+    {"playerleft",
+     {
+         {"walk_player_left", 0x10e2},
+         {"run_left", 0x10e2},
+         {"run_left", 0x10e2},
+         {"idle_stance", 0x2},
+         {"walk_start_left", 0x1040},
+     }},
+    {"playerright",
+     {
+         {"walk_player_right", 0x10e2},
+         {"run_right", 0x10e2},
+         {"run_right", 0x10e2},
+         {"idle_stance", 0x2},
+         {"walk_start_right", 0x1040},
+     }},
+    {"rocketback",
+     {
+         {"walk_rocket_back", 0xe2},
+         {"run_rocket_back", 0xe2},
+         {"run_rocket_back", 0xe2},
+         {"idle_rocket", 0x2},
+         {"walkst_rocket_back", 0x40},
+     }},
+    {"rocketleft",
+     {
+         {"walk_rocket_left", 0x10e2},
+         {"run_rocket_left", 0x10e2},
+         {"run_rocket_left", 0x10e2},
+         {"idle_rocket", 0x2},
+         {"walkst_rocket_left", 0x1040},
+     }},
+    {"rocketright",
+     {
+         {"walk_rocket_right", 0x10e2},
+         {"run_rocket_right", 0x10e2},
+         {"run_rocket_right", 0x10e2},
+         {"idle_rocket", 0x2},
+         {"walkst_rocket_right", 0x1040},
+     }}};
+
+AnimGroup AnimGroup::getBuiltInAnimGroup(AnimationSet &animations,
+                                         const std::string &name) {
+    auto findgroup = [&]() {
+        auto it =
+            std::find_if(kBuiltInAnimGroups.begin(), kBuiltInAnimGroups.end(),
+                         [&](const AnimGroup &g) { return g.name_ == name; });
+        if (it != kBuiltInAnimGroups.end()) {
+            return *it;
+        } else {
+            return kBuiltInAnimGroups[0];
```

Comment:
Looking at the input data it shouldn't ever ask for anything that doesn't exist. This is a failure case for sure.